client/data/courses/constants.ts

```diff
@@ -16,4 +17,14 @@ export const COURSE_DETAILS = {
 			i18n.translate( 'Set yourself up for blogging success' ),
 		],
 	},
+	[ COURSE_SLUGS.PAYMENTS_FEATURES ]: {
+		headerTitle: i18n.translate( 'Add Payments Features' ),
+		headerSubtitle: i18n.translate( 'Make Money on Your Website' ),
```

Comment:
This subtitle should also be in a "sentence case" format, and the final dot is also missing. If we change the title as suggested above, we could have a more generic subtitle here like "Watch our tutorial videos to get started.".  cc @sethmclellana8c
```suggestion
		headerSubtitle: i18n.translate( 'Watch our tutorial videos to get started.' ),
```